jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java

```diff
@@ -74,16 +72,9 @@
     /** The directory in the context to put the source files for the layer */
     private final String directoryInContext;
 
-    /** The extraction path in the image. */
-    private final AbsoluteUnixPath extractionPath;
-
-    private CopyDirective(
-        ImmutableList<LayerEntry> layerEntries,
-        String directoryInContext,
-        AbsoluteUnixPath extractionPath) {
```

Comment:
I think we might want to keep `extractionPath` here, or rename it, since `CopyDirectory` is meant to be a representation of the Docker `COPY` directive. In this current solution, though, `extractionPath` would always be `/`.